ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1441,29 +1441,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-upi
-- as: baremetalds-ipi-ovn-fips-ipv4-p2-f14
-  cron: 28 0 13,28 * *
+- as: baremetalds-ipi-ovn-ipv4-fips-p2-f14
```

Comment:
Can you also help update the ones in openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml and openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml

Reply:
Done